components/slider.json

```diff
@@ -0,0 +1,15 @@
+{
+    "slider": {
```

Comment:
is this component also in mobile? if not, please move this to component/platformcomponents/desktop